lambdas/check-existing-identity/src/main/java/uk/gov/di/ipv/core/checkexistingidentity/CheckExistingIdentityHandler.java

```diff
@@ -528,6 +544,23 @@ private void sendAuditEvent(
                         new AuditRestrictedDeviceInformation(deviceInformation)));
     }
 
+    @Tracing
+    private void sendVCsMigratedAuditEvent(
+            AuditEventTypes auditEventTypes,
+            AuditEventUser auditEventUser,
+            Vot vot,
+            String deviceInformation)
+            throws SqsException {
+        auditService.sendAuditEvent(
+                AuditEvent.createWithDeviceInformation(
+                        auditEventTypes,
+                        configService.getSsmParameter(ConfigurationVariable.COMPONENT_ID),
+                        auditEventUser,
+                        new AuditExtensionIdentityType(
+                                IdentityType.UPDATE, vot.equals(P0) ? null : vot),
```

Comment:
I think we need to agree what extensions should be included on this event - sorry I thought I'd marked the ticket as Blocked while we were working it out.

I don't think identity type really makes sense here - it's none of the three categories, it's migrating an existing identity. Instead, can we include the signatures of the VCs being recorded?

I'll update the ticket but I'm thinking something like:
```
"extensions": {
  "credentialSignatures": ["abc", "def"]
}
```

EVCS also records the signature of every VC stored, so we can cross-check between them to confirm that it's all working as expected.